akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -1848,13 +1850,29 @@ private[akka] class DDataShardCoordinator(
   override protected def unstashOneGetShardHomeRequest(): Unit = {
     if (getShardHomeRequests.nonEmpty) {
       // unstash one, will continue unstash of next after receive GetShardHome or update completed
-      val requestTuple = getShardHomeRequests.head
+      val requestTuple = {
```

Comment:
Could capture both nicely `val requestTuple @ (ref, _) = ...` instead of having to use the _1 syntax which is less clear.